### ISSUE TITLE
"Refresh Successful!" Edit in Loader

### ISSUE THREAD

**Issue by eduardo-munoz** [AdChain/AdChainRegistryDapp#205]
Please cut out everything underneath the `Refresh Successful!` text. Of course, keep the `CLOSE` button.

Since the green highlights both instances, it might be confusing to users. They'll be asking, "which one worked?". 

![image](https://user-images.githubusercontent.com/21111803/38773405-56e2b7e4-4000-11e8-84de-0bb5503976e2.png)

**Comment by samkim28** [comment]
Will remove it for Successful event. For In Progress, should we just reword it to be more simple? For example: You will receive 1 MetaMask prompt to update the status of this domain

**Comment by eduardo-munoz** [comment]
I see what you mean, and that works. Let's simplify it.
